index.js

```diff
@@ -370,7 +370,7 @@ async function main(provider, registrations, auctionRegistrations, registrations
   if (registeredDomainsAuction)
     console.log(`Registered domains with the auction registrar(legacy): ${registeredDomainsAuction}`);
 
-  if (!registeredDomainsAuction && auctionRegistrar)
+  if (!registeredDomainsAuction && auctionRegistrations && auctionRegistrations.length)
```

Comment:
Isn’t it like this?

```suggestion
  if (!registeredDomainsAuction && auctionRegistrations.length > 0)
```

Reply:
I think it should be the same validation  as here so it is consistent thats why I did it that way but im ok with putting the lenght >0 : https://github.com/rnsdomains/rns-suite/blob/f2fea19b7f4063d4a520b2b6cc15989e238ab3c3/index.js#L159